src/TenantDatabaseManagers/PermissionControlledMySQLDatabaseManager.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\TenantDatabaseManagers;
+
+use Stancl\Tenancy\Contracts\ManagesDatabaseUsers;
+use Stancl\Tenancy\DatabaseConfig;
+
+class PermissionControlledMySQLDatabaseManager extends MySQLDatabaseManager implements ManagesDatabaseUsers
+{
+    public static $grants = [
```

Comment:
I think it should be possible for a developer to overwrite grants. Some might to give even less grants and thereby more restrictive

Reply:
Since it's `public static`, it should be possible to do this, no?

```php
PermissionControlledMySQLDatabaseManager::$grants = [...];
```

Reply:
yes and no. It its not a singleton, then that will not work. 
I would suggest it to be in the config

Reply:
It's a static property, so it will work exactly like singletons.

Reply:
oh its static. sure that will work :D

Reply:
but.. is that intuitive ? 
I mean, if I don't want to go through the entire source code to understand that I can overwrite this. At least, that needs to be documented for people to get that

Reply:
Yeah, I'll add it to the docs. I also mentioned it in the config: https://github.com/stancl/tenancy/pull/382/files#diff-dd3b63ca90b9a9aac58961a8913d1f8fR199

Since these classes are optional, and you can use anything in place of them, I really don't like the idea of adding config keys for this stuff. The config should have things that are always in effect, rather than config for some classes that may not even be used. I will make a similar change to other things since I'm doing v3.